assets/js/modules/analytics/components/dashboard/LegacyAnalyticsDashboardWidgetTopPagesTable.js

```diff
@@ -74,13 +74,7 @@ const LegacyAnalyticsDashboardWidgetTopPagesTable = ( props ) => {
 					} );
 				} );
 				return (
-					<Link
-						href={ serviceURL }
-						external
-						inherit
-					>
-						{ title }
-					</Link>
+					<DetailsPermaLinks title={ title } path={ url } serviceURL={ serviceURL } />
```

Comment:
This looks good but please spread props onto their own line when more than 1 (or 2 very short).

Reply:
👍